resources/views/api/emails/voucher_history_email.blade.php

```diff
@@ -3,15 +3,15 @@
 @section('content')
 
     <section role="main">
-        <h1 style="text-align: center;">[xXx] Voucher Payment Records</h1>
+        <h1>Voucher Payment Records</h1>
 
         <p>Hi {{ $user }},</p>
 
-        <p>[xXx] You've requested a copy of {{ $trader }}'s voucher payment records for your reference, which is attached to this email.</p>
+        <p>You've requested a copy of {{ $trader }}'s voucher payment records for your reference, which is attached to this email.</p>
 
-        <p>[xXx] The file includes payment records from {{ $date }} @isset($max_date)to {{ $max_date }}. @endisset</p>
-        <p>[xXx] Total Vouchers: {{ count($vouchers) }}</p>
-        <p>[xXx] Total Value: £{{ count($vouchers) }}</p>
+        <p>The file includes payment records from {{ $date }} @isset($max_date)to {{ $max_date }}. @endisset</p>
+        <p>Total Vouchers: {{ count($vouchers) }}</p>
+        <p>Total Value: £{{ count($vouchers) }}</p>
 
         <p>The attached file is best viewed through a spreadsheet program such as Microsoft Excel, LibreOffice Calc or Google Sheets. If you have any problems with opening or downloading it, please email <a href="mailto:arc@neontribe.co.uk">arc@neontribe.co.uk</a>.</p>
```

Comment:
Yes - if you think it should go to other than admin (which will be set to something Anna gets on production).

We currently have MAIL_FROM_ADDRESS and MAIL_TO_ADMIN_ADDRESS

We don't generally direct mail back to devs from live apps, but might for the beta - so maybe MAIL_TO_DEVELOPER_TEAM might be appropriate if @charlesstrange2 thinks so too.

Yes - add to .env.example and config/mail.php